src/corehost/cli/fxr/hostpolicy_resolver.cpp

```diff
@@ -27,49 +29,27 @@ namespace
         trace::verbose(_X("--- Resolving %s version from deps json [%s]"), LIBHOSTPOLICY_NAME, deps_json.c_str());
 
         pal::string_t retval;
-        if (!pal::file_exists(deps_json))
-        {
-            trace::verbose(_X("Dependency manifest [%s] does not exist"), deps_json.c_str());
-            return retval;
-        }
 
-        pal::ifstream_t file(deps_json);
-        if (!file.good())
+        json_parser_t json;
+        if (!json.parse_file(deps_json))
         {
-            trace::verbose(_X("Dependency manifest [%s] could not be opened"), deps_json.c_str());
```

Comment:
What is the reasoning behind removing the logging?

Reply:
json_parser_t::parse_file() will print generic error messages (mentioning file names).  That seems sufficient.